client/layout/guided-tours/design-showcase-tour.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+	negate as not,
+} from 'lodash';
+import { isDesktop } from 'lib/viewport';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Continue,
+} from 'layout/guided-tours/config-elements';
+import {
+	inSection,
+	isNewUser,
+	isEnabled,
+	selectedSiteIsCustomizable,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const isAbTestInVariant = function() {};
```

Comment:
Oops! This isn't actually supposed to go into this PR. I want to wait for #8627 to be merged so I can use `isAbTestInVariant` from there. Will add a TODO here to reflect that.